templates/product.liquid

```diff
@@ -90,13 +90,13 @@
             {% endfor %}
           </select>
 
-          <span class="visually-hidden">{{ 'products.general.regular_price' | t }}</span>
+          <span id="ComparePriceA11y" class="visually-hidden">{{ 'products.general.sale_price' | t }}</span>
```

Comment:
Does this not result in the `products.general.sale_price` text being beside the product price and below `products.general.regular_price` is beside the compare at price.

This is the wrong way around, unless I'm missing something @matcaissy ?